backend/app/controllers/spree/admin/users_controller.rb

```diff
@@ -35,6 +35,10 @@ def create
       end
 
       def update
+        if params[:user][:password].blank? && params[:user][:password_confirmation].blank?
+          params[:user].delete(:password)
+          params[:user].delete(:password_confirmation)
+        end
```

Comment:
@gmacdougall I've thought about it, but a solution with changing `user_params` will be more complicated and less obvious. For example, we don't need this change for `create` action.
So in the `strong_parameters` way we need separate version of `user_params` for `update` action, but it will be too redundantly, imo.